packages/protocol/contracts/test/erc20/FailBridgeBackERC20.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.20;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+// An ERC20 token for creating a 'failed' status
+contract FailWhenBridgeBackCanonical is ERC20 {
```

Comment:
could we add some of the PR description into a comment here? just so we can remember the reason why for it.

it would also be nice to have full natspec compliance e.g. all public interfaces commented with their input / return params as well.

Reply:
@adaki2004 given that you are working on making the bridge more testable for failed message, maybe we simply skip this PR?